.github/workflows/database.yml

```diff
@@ -3,10 +3,14 @@ name: Database
 on:
   push:
     branches: [master]
+    paths-ignore:
+      - "doc/**"
```

Comment:
Let's only keep the ignore for "pull_request", so you can remove this one for "push" to master